### PR TITLE
Feature/preview

### PR DESCRIPTION
**Pull request by akileng56**
- Added  preview

### CODE REVIEW

File: src/AutoLoadMore/widget/AutoLoadMore.webmodeler.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Component, DOM } from "react";
+
+// tslint:disable-next-line:class-name
+export class preview extends Component<{}, {}> {
+    render() {
+        return DOM.div({});
```
Comment (Andries-Smit) [inline src/AutoLoadMore/widget/AutoLoadMore.webmodeler.ts]: Please add `className: "widget-auto-load-more-hide-preview"`

File: Gruntfile.js [hunk 2]
```diff
@@ -68,6 +74,15 @@ module.exports = function (grunt) {
             }
         },
 
+        file_append: {
+            addSourceURL: {
+                files: [{
```
Comment (Andries-Smit) [inline Gruntfile.js]: Make sure you have you linter enabled in JavaScript too.
Missing white space between `[ {` and `} ]`

File: Gruntfile.js [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
+    plugins: pluginsWidget
+});
+const pluginsPreview = webpackConfig[1].plugins.slice(0);
+pluginsPreview.push(new webpack.optimize.UglifyJsPlugin());
+Object.assign(webpackConfigRelease[1], webpackConfig[1], {
+    devtool: false,
+    plugins: pluginsPreview
 });
 
 module.exports = function (grunt) {
```
Comment (Andries-Smit) [inline Gruntfile.js:18]: missing file append in the watch

**Review by Andries-Smit — CHANGES_REQUESTED** [review]

File: Gruntfile.js (hunk 3 above)
Comment (akileng56) [reply Gruntfile.js:18]: Added

File: Gruntfile.js (hunk 2 above)
Comment (akileng56) [reply Gruntfile.js]: Done

File: src/AutoLoadMore/widget/AutoLoadMore.webmodeler.ts (hunk 1 above)
Comment (akileng56) [reply src/AutoLoadMore/widget/AutoLoadMore.webmodeler.ts]: Added

**Pull request merged**